browser/brave_content_browser_client.cc

```diff
@@ -218,9 +218,9 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "brave/browser/brave_ads/brave_ads_host.h"
 #endif  // BUILDFLAG(IS_ANDROID)
 
-#if BUILDFLAG(ENABLE_PLAYLIST)
+#if BUILDFLAG(ENABLE_PLAYLIST) && !BUILDFLAG(IS_ANDROID)
```

Comment:
can we add a new buildflag enable_playlist_webui or something?

Reply:
Sounds good! Done.